### PR TITLE
Add initial attempt at travis support

### LINKED ISSUE #47: Add Continuous Integration
#42 adds a script to test the modules. We should add some CI.

@ucbjrl, should we not use travis for new things? What are your thoughts?

### PR DESCRIPTION
**Pull request by grebe**
Closes #47.

@ucbjrl do you have the required permissions to enable travis for this repo?

### CODE REVIEW

**Comment by ucbjrl** [comment]
Let's see ...

**Comment by ucbjrl** [comment]
Looks like I can. Let's see if I can trigger a build ...

**Comment by ucbjrl** [comment]
Travis complains about lack of a config file in master, and I can't seem to get it to see other branches. I'll try a gratuitous change to this PR to see if that triggers a build. If not, we may need to commit this (so master has a config).

**Comment by grebe** [comment]
OK, I have fixes to make but this looks great. Thanks so much, Jim!

File: .travis_install.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ bash miniconda.sh -b -p $HOME/miniconda
 export PATH="$HOME/miniconda/bin":$PATH
 conda config --set always_yes yes --set changeps1 no
 conda update -q conda
-conda install jupyter
+conda install jupyter typing
```
Comment (edwardcwang) [inline .travis_install.sh]: Hmm, interesting that Travis' Python3 doesn't come with `typing` by default

Comment (grebe) [reply .travis_install.sh]: It's being aliased by miniconda, probably. I guess I could use pip to install jupyter, but I like conda better

Comment (grebe) [reply .travis_install.sh]: Or maybe not. It looks like my path isn't being updated...

File: runtest.py [hunk 2]
```diff
@@ -33,6 +33,11 @@ def _notebook_run(path):
     return nb, errors
 
 def check_errors(expected, actual):
+    if len(expected) != len(actual):
+        print("len(expected) = " + str(len(expected)) + "\n")
```
Comment (edwardcwang) [inline runtest.py]: Woot

Comment (grebe) [reply runtest.py]: lol is there a less dumb way to do this? python string interpolators are still new, right?

Comment (edwardcwang) [reply runtest.py]: Yeah I think f-interpolators are 3.6+ only :-(

The woot was mainly for the fact that tests now run :-)

**Comment by chick** [comment]
This has been superseded by #136

**Pull request closed without merging**